core/src/main/java/tech/tablesaw/api/DoubleColumn.java

```diff
@@ -109,19 +109,27 @@ public static DoubleColumn create(final String name, final long[] arr) {
   }
 
   public static DoubleColumn create(final String name, final List<Number> numberList) {
-    final double[] doubles = new double[numberList.size()];
-    for (int i = 0; i < numberList.size(); i++) {
-      doubles[i] = numberList.get(i).doubleValue();
+    DoubleColumn newColumn = new DoubleColumn(name, new DoubleArrayList(0));
+    for (Number number : numberList) {
+      if (number == null) {
+        newColumn.appendMissing();
+      } else {
+        newColumn.append(number.doubleValue());
```

Comment:
It might be easier to make `append` deal with `null`s. Then you wouldn't have to update multiple `create` methods. And it'd also do the right thing if the user calls `append(null)`

Reply:
the change was put in because the original version copied the objects to primitive arrays so it would NPE, but that's a good point. Append should handle nulls